_templates/sidebar_links.html

```diff
@@ -75,7 +71,17 @@
     href="https://github.com/ipython/ipython/wiki/Dev:-Index">Development information</a></li>
     <li><a class="reference external" href="http://travis-ci.org/#!/ipython/ipython">
         Travis CI</a></li>
-  </ul>          
+  </ul>
+</div>
+<div class="sidebarblock">
```

Comment:
Let's try removing gitter for now, and put reddit at the bottom of the 'community' section. I'd also put the wiki under developer information rather than community.

None of this is set in stone - we can easily put the gitter link back later if we change our minds.